From 4b662f8be39fba9d7769cd980bae886cce10d72c Mon Sep 17 00:00:00 2001
From: Richard Zhu <hongxing.zhu@nxp.com>
Date: Fri, 15 Jun 2018 13:31:34 +0800
Subject: [PATCH 4024/5242] MLK-18603 ARM64: dts: imx8mm: reboot fail when
 boot up with -m4 dtb

commit  ed143b50adc5821f09bb3abdeed4470eb0dcb280 from
https://source.codeaurora.org/external/imx/linux-imx.git

The wdog is mandatory required by the reboot.
Don't disable it in -m4 dts.

Signed-off-by: Richard Zhu <hongxing.zhu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../arm64/boot/dts/freescale/fsl-imx8mm-evk-m4.dts |    4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk-m4.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk-m4.dts
index cd5abda..2501937 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk-m4.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk-m4.dts
@@ -48,7 +48,3 @@
 &uart4 {
 	status = "disabled";
 };
-
-&wdog1{
-	status = "disabled";
-};
-- 
1.7.9.5

